### ISSUE TITLE
English version of the research paper

### ISSUE THREAD

**Issue by ziedbouf** [filippocastelli/GPnet#1]
Hi filippo,  Thanks for the published work related to the guassian process classification on networks. is there please version of your paper in english as i am interested on the same techniques applied for networks.

**Comment by filippocastelli** [comment]
Hi! I'd be happy to help you, can I please ask you to contact me at castelli@lens.unifi.it ?

**Comment by ziedbouf** [comment]
Yes sure thanks @filippocastelli  and i am really interested on you research. Do you have any roadmap for this project.
